### PR TITLE
feat: add color to progress mark and bar

### PR DESCRIPTION
**Pull request by qweeah**
**What this PR does / why we need it**:
This PR adds color to the TTY output.
- The spinner will be renderred in light yellow
- The done mark will be renderred in light green
- The progress bar will be renderred in light blue

image

**Please check the following list**:
- [ ]  Does the affected code have corresponding tests, e.g. unit test, E2E test?
- [ ]  Does this change require a documentation update?
- [ ]  Does this introduce breaking changes that would require an announcement or bumping the major version?
- [ ]  Do all new files have an appropriate license header?

### CODE REVIEW

File: cmd/oras/internal/display/status/progress/status.go [hunk 1]
```diff
@@ -35,6 +35,12 @@ const (
 	zeroDigest   = "  └─ loading digest..."
 )
 
+var (
+	spinnerColor  = aec.LightYellowF
+	doneMarkColor = aec.LightGreenF
+	progressColor = aec.LightBlueB
```
Comment (shizhMSFT) [inline cmd/oras/internal/display/status/progress/status.go:41]: Is the `LightBlueB` visible in Windows PowerShell?

File: cmd/oras/internal/display/status/progress/status.go [hunk 2]
```diff
@@ -128,13 +134,15 @@ func (s *status) String(width int) (string, string) {
 	lenLeft := 0
 	if !s.done {
 		lenBar := int(percent * barLength)
-		bar := fmt.Sprintf("[%s%s]", aec.Inverse.Apply(strings.Repeat(" ", lenBar)), strings.Repeat(".", barLength-lenBar))
+		bar := fmt.Sprintf("[%s%s]", progressColor.Apply(strings.Repeat(" ", lenBar)), strings.Repeat(".", barLength-lenBar))
 		speed := s.calculateSpeed()
-		left = fmt.Sprintf("%c %s(%*s/s) %s %s", s.mark.symbol(), bar, speedLength, speed, s.prompt, name)
+		left = fmt.Sprintf("%s %s(%*s/s) %s %s",
+			spinnerColor.Apply(fmt.Sprintf("%c", s.mark.symbol())),
```
Comment (shizhMSFT) [inline cmd/oras/internal/display/status/progress/status.go:140]:
```suggestion
			spinnerColor.Apply(string(s.mark.symbol())),
```

File: cmd/oras/internal/display/status/progress/status.go (hunk 1 above)
Comment (qweeah) [reply cmd/oras/internal/display/status/progress/status.go:41]: PS by default uses dark blue I think

image

Comment (qweeah) [reply cmd/oras/internal/display/status/progress/status.go:41]: From my view, the bar can be seen clearly
image

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**